helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -39,6 +39,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     DB_SSL_MODE: "verify-full"
     HMPPS_SQS_USE_WEB_TOKEN: true
+    FEATURE_PROBATION_LIVE: false
 
```

Comment:
Wonder about naming here, as a probation journey is already live (even though using VLB as the type). FEATURE_MASTER_VLPM_TYPES?  Meh.

Reply:
Done